### PR TITLE
Fix Logging False-Positive logs

### PR DESCRIPTION
**Pull request by shadinaif**
**TL;DR -** Fix Logging False-Positive logs caused by this package and raised in `edx-submission` package

JIRA: 

**What changed?**

- [ More in depth breakdown of changes ]
- [ Peripheral things that got changed ]
- [ etc... ]

**Developer Checklist**

- [ ] Reviewed the [release process](https://github.com/edx/edx-ora2/blob/master/.github/release_process.md)
- [ ] Translations and JS/SASS compiled
- [ ] Bumped version number in [setup.py](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/setup.py#L39) and [package.json](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/package.json#L3)

**Testing Instructions**

[ How should a reviewer test this PR? ]

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @openedx/content-aurora

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @shadinaif! I've created [OSPR-6503](https://openedx.atlassian.net/browse/OSPR-6503) to keep track of it in JIRA, where we prioritize reviews. Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by shadinaif** [comment]
@OmarIthawi your review please 😁

**Review by OmarIthawi — COMMENTED** [review]
@shadinaif thanks. I think we should add two items to this pull request:

 - Details steps to reproduce the bug/log report and their expected vs. actual behaviour.
 - Automated tests to test the newly added lines.

**Comment by natabene** [comment]
@shadinaif Thank you for the contribution. Is this ready for a review?

**Comment by natabene** [comment]
@shadinaif Can you let me know once you implement the feedback requested?

**Comment by shadinaif** [comment]
Thank you @natabene I'll address all tomorrow 👍🏼

**Comment by natabene** [comment]
@shadinaif Is this ready for our review?

**Comment by natabene** [comment]
@shadinaif We have not heard from you in a while, so I will close this for now. Please feel free to re-open should you decide to pursue this again.

**Comment by shadinaif** [comment]
Is there a way to reopen the PR @OmarIthawi or should I create a new one?

**Comment by shadinaif** [comment]
Thank you @natabene @OmarIthawi !
